homeassistant/helpers/event.py

```diff
@@ -581,6 +581,11 @@ def async_setup(self) -> None:
 
         self._last_info = self._info.copy()
         self._create_listeners()
+        _LOGGER.debug(
+            "Template group %s listens for %s",
+            [template.template for template in self._track_templates],
```

Comment:
We would create a list each time just to throw away

Reply:
These never change so I guess we could store it.

Reply:
Can't we just put in `self._track_templates` and add a `__repr__` to templates

Reply:
Turns out I can just log `self._track_templates` and get the desired result 🤷